script.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function() { 
+    $('#action').click(function() {
+
+    var letterGrade = $('#letter-grade').val().toUpperCase();
+    var myFavoriteThings = [];
+        myFavoriteThings.push("Photography");   
+        myFavoriteThings.push("Graphic Design"); 
+        myFavoriteThings.push("Fashion"); 
+        myFavoriteThings.push("Dancing"); 
+
+    //Write your decision structure between this comment...
+    if (letterGrade == 'A') { 
+        $("#decision-holder").append ("Grade range for A: 90% to 100%")  
```

Comment:
By using .append() here - the text will be added each time. If you want the form to change what displays here - .text() would be a better choice.